global.R

```diff
@@ -42,11 +42,9 @@ package.check <- lapply(
   }
 )
 
-# Install riskmetric from github and load it
-if(!require(riskmetric)){
-  devtools::install_github("pharmaR/riskmetric")
-  library(riskmetric)
-}
+# Install a specific version of riskmetric from GitHub and load it.
+devtools::install_github("pharmaR/riskmetric", ref = "release-v0.1.1")
```

Comment:
So, this line of code will re-install the `0.1.1` version every time you run the app. Is that the intended behavior? How about we use an if-then statement using packageVersion("riskmetric") to see if the user has the correct version. We can also add this version to the `DESCRIPTION` file.

![image](https://user-images.githubusercontent.com/17878953/150000335-a357ce04-2584-4226-8be1-4ee8c4877561.png)

Reply:
Change this line by `library(riskmetric)`. The package version of the app will take care of package versioning.

Reply:
Hi @AARON-CLARK, I am having trouble pushing the updated `global.R file`. My local git doesn't know how to reconcile my changes with the branches that are now part of this PR. 

Could you please undo the changes I made? Basically remove lines 46-47 and add back 45-49?

Reply:
done!